tests/system/action/conftest.py

```diff
@@ -0,0 +1,2 @@
+def pytest_runtest_logreport(report):  # type:ignore
+    report.nodeid = ">" + report.nodeid[-75:]
```

Comment:
What is this for?

Reply:
Removed
I use it for shifting the pytest output to see the errors better and not only the initial path